internal/testing/e2e/common.go

```diff
@@ -60,6 +62,7 @@ func New(t *testing.T) (OM, func()) {
 
 // RunMain provides the setup and teardown for Open Match e2e tests.
 func RunMain(m *testing.M) {
+	resolver.SetDefaultScheme("passthrough")
```

Comment:
We need to add a comment here stating the reason for this so in future, this is not cryptic to someone who does not understand the underlying issue.